docs/csharp/language-reference/keywords/file.md

```diff
@@ -1,18 +1,17 @@
 ---
-description: "file access modifier: Declare types whose scope is the file in which it's declared"
+description: "file modifier: Declare types whose scope is the file in which it's declared"
 title: "file keyword - C# Reference"
 ms.date: 09/15/2022
 f1_keywords: 
   - "file_CSharpKeyword"
-  - "file"
 helpviewer_keywords: 
   - "file keyword [C#]"
 ---
 # file (C# Reference)
 
-Beginning with C# 11, the `file` contextual keyword is a type access modifier.
+Beginning with C# 11, the `file` contextual keyword is a type modifier.
 
-File-scoped access restricts a top-level type's scope and visibility to the file in which it's declared. The `file` accessor modifier will generally be applied to types written by a source generator. File scoped types provide source generators with a convenient way to avoid name collisions among generated types. The `file` access modifier declares a file-scoped type, as in this example:
+The `file` modifier restricts a top-level type's scope and visibility to the file in which it's declared. The `file` modifier will generally be applied to types written by a source generator. File-scoped types provide source generators with a convenient way to avoid name collisions among generated types. The `file` modifier declares a file-scoped type, as in this example:
```

Comment:
```suggestion
The `file` modifier restricts a top-level type's scope and visibility to the file in which it's declared. The `file` modifier will generally be applied to types written by a source generator. File-local types provide source generators with a convenient way to avoid name collisions among generated types. The `file` modifier declares a file-local type, as in this example:
```